lib/core/src/persist/mod.rs

```diff
@@ -268,7 +268,7 @@ impl Persister {
                 ptx.tx_id NOT IN (SELECT refund_tx_id FROM chain_swaps WHERE refund_tx_id NOT NULL)
             AND {}
             ORDER BY                             -- Order by swap creation time or tx timestamp (in case of direct tx)
-                COALESCE(rs.created_at, ss.created_at, cs.created_at, ptx.timestamp) DESC
+                COALESCE(rs.created_at, ss.created_at, cs.created_at, ptx.timestamp, strftime('%s', 'now')) DESC
```

Comment:
Would it make sense to store the "seen" time (current time) in the db in this case if there is no tx timestamp? Otherwise won't these payments bubble up the payment list

Reply:
Makes sense but I will need to make sure we don't override the timestamp as we are doing "insert or replace" for tx_data.
I will take a look.

Reply:
Done